litellm/utils.py

```diff
@@ -1095,7 +1095,7 @@ def pre_call(self, input, api_key, model=None, additional_args={}):
             curl_command += (
                 f"{formatted_headers} \\\n" if formatted_headers.strip() != "" else ""
             )
-            curl_command += f"-d '{str(data)}'\n"
+            curl_command += f"-d '{json.dumps(data)}'\n"
```

Comment:
two requests 
- can we `try/except` the `json.dumps` so it does not block the LLM call 
- can you modify this to not use an `f` string. in python `f` strings are eagerly evaluated and can impact latency